googleApiCrawl.py

```diff
@@ -0,0 +1,41 @@
+import os
+import googlemaps
+import json
+import requests
+from os.path import join, dirname
+from dotenv import load_dotenv, find_dotenv
+
+dotenv_path = join(dirname(__file__), '.env')
+load_dotenv(find_dotenv(), override=True)
+
+API_KEY = os.environ.get("API_KEY")
+
+gmaps = googlemaps.Client(key=API_KEY)
+
+# Twin Peaks is center of SF : 37.755305, -122.466763
+gmaps_result = gmaps.places_radar(location=(37.755305, -122,466763)
+                                  , radius=50000
```

Comment:
Is it common to use comma in front of the variable? Please double check on the python style guides